webpages/settings/components/addon-group-header.html

```diff
@@ -6,38 +6,38 @@
 </template>
 
 <style>
-  .addon-group.margin-above {
-    margin-top: 20px;
-  }
-
   .addon-group {
     display: flex;
     align-items: center;
     cursor: pointer;
     margin: 0px 11px;
     padding: 0px 10px;
-  }
 
-  .addon-group > img {
-    height: 24px;
-    width: 24px;
-    margin-inline-end: 9px;
-    padding: 4px;
-    border-radius: 4px;
-    filter: var(--content-icon-filter);
-    transition: background-color 0.2s ease;
-  }
+    &.margin-above {
+      margin-top: 20px;
+    }
 
-  .addon-group:hover > img {
-    background-color: var(--hover-highlight);
-  }
+    & > img {
```

Comment:
Personally I find it quite confusing to know when the ampersand is needed and when it's not. Since Chrome 120+ and Firefox 117+ it is needed in fewer places, see: https://developer.chrome.com/blog/css-nesting-relaxed-syntax-update